src/main/scala/com/brigade/finagle/consul/catalog/ConsulCatalogResolver.scala

```diff
@@ -0,0 +1,104 @@
+package com.brigade.finagle.consul.catalog
```

Comment:
So this class is mean to be used to help a service find other services registered in Consul by calling `#addrOf` with the...